src/services/room-service.js

```diff
@@ -276,6 +374,59 @@ export default class RoomService {
     });
   }
 
+  async deleteRoomInvitation({ room, invitation }) {
+    await this.roomInvitationStore.deleteRoomInvitationById(invitation._id);
+    const remainingRoomInvitations = await this.getRoomInvitations(room._id);
+
+    return remainingRoomInvitations;
+  }
+
+  // ROOM MESSAGES
+
+  async createRoomMessage({ room, text, emailNotification }) {
+    const messages = cloneDeep(room.messages);
+    messages.push({
+      key: uniqueId.create(),
+      text,
+      emailNotification,
+      createdOn: new Date()
+    });
+
+    await this.roomStore.updateRoomMessages(room._id, messages);
+
+    const updatedRoom = await this.roomStore.getRoomById(room._id);
+
+    return updatedRoom;
+  }
+
+  async updateRoomMetadata(roomId, { name, slug, isCollaborative, description }) {
```

Comment:
Just noticing that this method belongs in the `// ROOMS` section of the module.

Reply:
True, I moved it (and removed the header comments in this file).